test/specs/sortingtest.e2e.js

```diff
@@ -0,0 +1,35 @@
+const { pages } = require('../pages/Pages');
+const { parsePrice } = require('../helpers/parsePrise');
+const { getAllTextContents } = require('../helpers/getAllTextContents');
+
+
+
+
+describe('Sorting dropdown', () => {
+    before(async () => {
+        await pages.loginPage.navigate();
+        await pages.loginPage.performLogin('standard_user', 'secret_sauce');
+
+    })
+
+    const selectOptions = [
+        { option: "az", sort: (elements) => elements.slice().sort() },
+        { option: "za", sort: (elements) => elements.slice().sort((a, b) => a < b ? 1 : -1) },
+        { option: "lohi", sort: (elements) => elements.slice().sort((a, b) => parsePrice(a) - parsePrice(b)) },
+        { option: "hilo", sort: (elements) => elements.slice().sort((a, b) => parsePrice(b) - parsePrice(a)) }
+    ];
+
+
+    it('Test sorting Dropdown option', async () => {
```

Comment:
The below code is really cool!
Let's make everything even better.

```suggestion
selectOptions.forEach(selOption => 
    it('Test sorting Dropdown option ' + selOption.option, async () => {
```
This will create 4 tests instead of only one and will make it easier to identify what test failed if any